api_db.php

```diff
@@ -94,15 +94,28 @@ function SQLite3_connect($trytoreconnect)
 	{
 		$limit = "WHERE timestamp <= ".$_GET["until"];
 	}
-	$results = $db->query('SELECT client,count(client) FROM queries '.$limit.' GROUP by client order by count(client) desc limit 10');
+	$results = $db->query('SELECT client,count(client) FROM queries '.$limit.' GROUP by client order by count(client) desc limit 20');
 
 	$clients = array();
+	$num = 0;
 
 	if(!is_bool($results))
-		while ($row = $results->fetchArray())
+		while (($row = $results->fetchArray()) && $num < 10)
 		{
-			$clients[$row[0]] = intval($row[1]);
-			// var_dump($row);
+			// Convert client to lower case
+			$c = strtolower($row[0]);
+			if(array_key_exists($c, $clients))
+			{
+				// Entry already exists, add to it (might appear multiple times due to mixed capitalization in the database)
+				$clients[$c] += intval($row[1]);
+			}
+			else
+			{
+				// Entry does not yet exist
+				$clients[$c] = intval($row[1]);
+				// Increase number of clients
+				$num++;
```

Comment:
We should go through every returned result and condense, then sort, then limit to 10. If we limit first, then we could have even more inaccurate results.